webtransport/quic/handlers/client-indication.quic.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+import logging
+
+from aioquic.asyncio import QuicConnectionProtocol
+from aioquic.quic.events import QuicEvent
+from typing import Dict
+
+async def notify(connection):
```

Comment:
There should be two blank lines before a top-level def according to PEP8: https://www.python.org/dev/peps/pep-0008/#blank-lines.

Also, should `connection` have a type specified?

Also `notify` is not a very clear name. How about `notify_pass`?

Reply:
Done.